### ISSUE TITLE
ドキュメントの整理

### ISSUE THREAD

**Issue by ghost** [emacs-jp/issues#27]
vimのhttp://vim-jp.org/vimdoc-ja/usr_41.htmlこんなページで開発者者支援側のまとまったドキュメントがあるといいなとおもいます。
1本のまとまる場所が欲しいです。

**Comment by syohex** [comment]
そうですね. ドキュメントを書くのはなかなかの労力なので全然行えていません. PRはガンガン受け付けるので, 何かあればよろしくお願いします. このようなドキュメントが欲しい等でも構いません.
